mods/tuxemon/db/technique/boulder.json

```diff
@@ -11,7 +12,7 @@
   "is_fast": false,
   "potency": null,
   "power": 1.0,
-  "range": "special",
+  "range": "ranged",
```

Comment:
boulder is a "special" technique because it doesn't apply damage, and instead applies the effect "hardshell".  this move in particular always comes up because its assumed to be an attack.  https://wiki.tuxemon.org/index.php?title=Boulder

Reply:
Ah, I see. It looks like we don't yet have an implementation of the "hardshell" effect. I can stub this out and add a TODO to implement the effect logic.